src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -301,13 +303,23 @@
             importPJNZ: mapActionByName("baseline", "importPJNZ"),
             importShape: mapActionByName("baseline", "importShape"),
             importPopulation: mapActionByName("baseline", "importPopulation"),
+            deleteUploadInputs: mapActionByName("baseline", "deleteAll"),
             importSurvey: mapActionByName("surveyAndProgram", "importSurvey"),
             importProgram: mapActionByName("surveyAndProgram", "importProgram"),
             importANC: mapActionByName("surveyAndProgram", "importANC"),
+            deleteReviewInputs: mapActionByName("surveyAndProgram", "deleteAll"),
             async importDataset() {
                 this.stopPolling();
 
                 this.loading = true;
+
+                if (this.hasShapeFile) {
```

Comment:
Trying to avoid sending unnecessary delete action here but I think it might be better to ignore this checking, your thoughts? 

Perhaps I could check if surveyFile exists too `if(shape || survey)...`

Reply:
See comment above, I think you only need to send action to delete baseline files. You could check if any exist before sending that action i.e. any of shape, pjnz or population